frontend/src/views/Transactions/Transactions.jsx

```diff
@@ -0,0 +1,185 @@
+import BCAlert from '@/components/BCAlert'
+import BCButton from '@/components/BCButton'
+import { DownloadButton } from '@/components/DownloadButton'
+import { ROUTES } from '@/constants/routes'
+import { useApiService } from '@/services/useApiService'
+import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Box, Typography } from '@mui/material'
+import { AgGridReact } from 'ag-grid-react'
+import * as dayjs from 'dayjs'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { dummy } from './data'
+import { getStatus } from '@/utils/getStatus'
+import { getOrgnization } from '@/utils/getOrganization'
+
+export const Transactions = () => {
+  const navigate = useNavigate()
+  const apiService = useApiService()
+
+  const [isDownloadingTransactions, setIsDownloadingTransactions] =
+    useState(false)
+  const [alertMessage, setAlertMessage] = useState('')
+  const [alertSeverity, setAlertSeverity] = useState('info')
+
+  const handleDownloadTransactions = async () => {
+    alert('Download Transactions clicked')
+
+    // enable once we have transactions in db
+
+    // setIsDownloadingTransactions(true)
+    // setAlertMessage('')
+    // try {
+    //   await apiService.download('/transactions/export')
+    //   isDownloadingTransactions(false)
+    // } catch (error) {
+    //   console.error('Error downloading organization information:', error)
+    //   isDownloadingTransactions(false)
+    //   setAlertMessage('Failed to download organization information.')
+    //   setAlertSeverity('error')
+    // }
+  }
+
+  return (
+    <>
+      <div>
+        {alertMessage && (
+          <BCAlert data-test="alert-box" severity={alertSeverity}>
+            {alertMessage}
+          </BCAlert>
+        )}
+      </div>
+      <Typography variant="h5" mb={2}>
+        Transactions
+      </Typography>
+      <Box display={'flex'} gap={2} mb={2}>
+        <BCButton
+          variant="contained"
+          size="small"
+          color="primary"
+          startIcon={<FontAwesomeIcon icon={faCirclePlus} />}
+          onClick={() => navigate(ROUTES.TRANSACTIONS_ADD)}
+        >
+          <Typography variant="subtitle2">New Transfer</Typography>
```

Comment:
I think this should be "New Transaction". Can you please refer https://preview.uxpin.com/59145260c7d8cc48969ac79401cc03d84843c3d9#/pages/165270334/simulate/sitemap
and update accordingly.

Reply:
changed to 'New Transaction'